proposals/lip-0051.md

```diff
@@ -2540,6 +2540,8 @@ The following steps are executed as part of the execution of a cross-chain updat
 def verifyCrossChainMessage(trs: Transaction, ccm: CCM) -> None:
     # This is getTokenIDLSK() for channels with the mainchain.
     messageFeeTokenID = Interoperability.getMessageFeeTokenID(ccm.sendingChainID)
+    if messageFeeTokenID != getTokenIDLSK():
```

Comment:
This check should be cast only on mainchain, as on a sidechain the message fee can be anything (it must always be a native token)